### ISSUE TITLE
Tab groups inaccessible with 0.9.6 on FF 35.0.1

### ISSUE THREAD

**Issue by chludwig-haufe** [char101/tabgroupsmenu#44]
Hi,

it seems that the latest version of the add-on broke tab groups for me on Ubuntu 14.04 / FF 35.0.1. The menu item does not appear any more and when I click on the tab group button in the tool bar nothing happens, even though I have multiple tab groups.

When I remove the add-on, then a click on the tab groups button has the expected effect again. If I then install the add-on, I see the menu item - but after the next restart, the menu item has disappeared again and a click on the tab groups button again shows no effect at all.

Regards
Christoph

**Comment by nogo** [comment]
I have the same problem, after restart their is no chance to access the tabgroup menu or the panorama button.

**Comment by char101** [comment]
Tested with Firefox 35 with clean profile, I do not experience any problem.

![add-ons manager - mozilla firefox](https://cloud.githubusercontent.com/assets/71255/6164353/edae72d2-b2d0-11e4-80fe-4b6a5c6f11fb.png)

**Comment by char101** [comment]
I could be platform (OS) related, although the code has no platform specific part.

**Comment by chludwig-haufe** [comment]
Let me know if there are any data points, dumps, whatsoever that can help narrow down the cause of this issue. I will try to provide as much information as I can.

**Comment by char101** [comment]
You can

* open the browser console (ctrl + shift + j), copy the contents, paste into a text editor, find the lines that contains tabgroupsmenu, then paste those lines here
* in the meantime, downgrade to a lower version here https://addons.mozilla.org/en-us/firefox/addon/tabgroups-menu/versions/

**Comment by char101** [comment]
Never mind with the browser console, I can reproduce the case with firefox in Arch linux.

**Comment by nogo** [comment]
So the problem ist os related? I forgot to say that I'm on linux (arch) to.

**Comment by char101** [comment]
Seems so. No problem on Windows. And only the first window is affected, if you open another window the addon works fine. It seems there is a bug with eager panorama initialization on linux. I suggest that you use version 0.9.2 for now.

**Comment by SandJ** [comment]
Not just me then.  Firefox 35.0.1 and Xubuntu 14.04

My tab groups (I have about 60) became inaccessible a few days ago.  If I create a new Window, it works in that new window.  If I cause a crash with Firefox running, the tab groups are working while it says "Restore Session?" and continues to work after I have restored all the tabs.  However, if I close Firefox and restart it they stop working again.

I tried disabling all the add-ons, no difference.

I have purged my cache, bookmarks and most of my history.  No difference.

**Comment by Secret-Code** [comment]
Issue exists on Windows as well - I have the same symptoms on Windows 7 as well as Kubuntu 14.04. The dropdown button and menu bar item intermittently (but nearly always) don't appear on the first window created, but do in a new window. Browser console contains no lines with 'tabgroupsmenu' in. Have 0.9.6 - downgrading to 0.9.2 works around the issue - thanks.

**Comment by SandJ** [comment]
Yesterday I replaced tabgroupsmenu@char.cc.xpi in ~/.mozilla/firefox/wgabmngv.default/extensions with a backup I made in December.  Today it is working again as it was replaced at 7:23 this morning, presumably through an automatic Firefox update.  The most recent file in it is dated 14/1/15 8:31 and so that must be this version:
https://github.com/char101/tabgroupsmenu/commit/24caeb650e700c4d35380701a591828ad394a6ff
I guess the latest changes have been rolled back.

**Comment by char101** [comment]
Yes, I have disabled the newer versions until I have the time (motivation) to rewrite the addon.

**Comment by pan64** [comment]
It looks like I have the same issue too. FF 36.0.1, TabGroupsMenu 0.9.2, ubuntu 12.04. TabGroups menu does nothing when I start FF. I need to disable it, restart FF and reenable (and it will work as long as FF running).
